migrations/capcons/capabilitymigration.go

```diff
@@ -67,65 +70,107 @@ func (m *CapabilityValueMigration) Migrate(
 	storageKey interpreter.StorageKey,
 	_ interpreter.StorageMapKey,
 	value interpreter.Value,
-	_ *interpreter.Interpreter,
+	inter *interpreter.Interpreter,
 	_ migrations.ValueMigrationPosition,
 ) (
 	interpreter.Value,
 	error,
 ) {
 
+	// Migrate path capabilities to ID capabilities
+	if pathCapabilityValue, ok := value.(*interpreter.PathCapabilityValue); ok { //nolint:staticcheck
+		return m.migratePathCapabilityValue(pathCapabilityValue, storageKey, inter)
```

Comment:
Diff looks a bit odd at the bottom of the file, but all that changed is refactoring the switch, which only has single case, into a separate function.